src/components/tabset/tabset.component.html

```diff
@@ -15,7 +15,28 @@
         [class.disabled]="tab.disabled"
         [ngClass]="tab.customClass">
 
-        <a class="nav-link"
+       <ng-container *ngIf="tab.routerPath">
+            <a class="nav-link"
+            [id]="tab.id"
+            role="tab"
+            uxTabbableListItem
+            uxFocusIndicator
+            (mousedown)="_tabset.select(tab)"
+            (activated)="_tabset.select(tab)"
```

Comment:
If there's a routerLink attached, I don't think we do any of this. I think it should just be the link.